test/image/strict-d3.js

```diff
@@ -25,6 +41,14 @@ selProto.style = function() {
     return originalSelStyle.apply(sel, arguments);
 };
 
+function checkAttrVal(sel, key) {
+    // setting the transform attribute on a <clipPath> does not
+    // work in Chrome, IE and Edge
+    if(sel.node().nodeName === 'clipPath' && key === 'transform') {
```

Comment:
No need for additional jasmine tests cases, this here made many animation and interaction test fail.